sdk/servicebus/service-bus/src/receivers/receiverCommon.ts

```diff
@@ -245,23 +246,25 @@ export function settleMessageOperation(
     throw error;
   }
 
-  // Message Settlement with managementLink
-  // 1. If the received message is deferred as such messages can only be settled using managementLink
-  // 2. If the associated receiver link is not available. This does not apply to messages from sessions as we need a lock on the session to do so.
-  if (isDeferredMessage || ((!receiver || !receiver.isOpen()) && !isDefined(message.sessionId))) {
-    return context
-      .getManagementClient(entityPath)
-      .updateDispositionStatus(message.lockToken!, operation, {
-        ...options,
-        associatedLinkName,
-        sessionId: message.sessionId
-      })
-      .catch((err) => {
-        throw translateServiceBusError(err);
-      });
-  }
+  try {
+    // Message Settlement with managementLink
+    // 1. If the received message is deferred as such messages can only be settled using managementLink
+    // 2. If the associated receiver link is not available. This does not apply to messages from sessions as we need a lock on the session to do so.
+    if (isDeferredMessage || ((!receiver || !receiver.isOpen()) && !isDefined(message.sessionId))) {
+      return context
+        .getManagementClient(entityPath)
+        .updateDispositionStatus(message.lockToken!, operation, {
+          ...options,
+          associatedLinkName,
+          sessionId: message.sessionId
+        });
```

Comment:
Do thrown errors still get translated? Wondering why that code path was removed.

Edit: I see you have this inside a try/catch, but if `updateDispositionStatus` returns a promise then you won't be catching it because you don't await it.

Reply:
I see. 
Added awaits and wrapped around with try-catch and translating with translateServiceBusError.

Reply:
Let me know if that looks good.